### PR TITLE
WC: Toasts

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
## Toasts
Added toasts throughout the app using the [react-toastify package](https://www.npmjs.com/package/react-toastify), 

### Other Changes
- Added logout button 
- Added Favicon for the web app
- Converted .png home page background image to .webp for reducing size
- Fixed video remove from playlist functionality
- Added a dynamic button on the home page to navigate to the login page when unauthenticated, and to the playlist's page otherwise.
- Navigating back to user history on login/ signup using useLocation() hook.

### Major files to check
- [ ] API calls
- [ ] Auth Pages

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean , Learned how to implement  the toast

**Comment by Prabhav-Sharma** [comment]
> Code is Clean , Learned how to implement the toast

I learned from someone's PR as well ^_^.

File: src/pages/Auth/Login.jsx [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
-      {
-        email,
-        password,
-      },
-      setLoginToggle,
-      authDispatch
-    );
+    await login({ email, password }, setLoginToggle, authDispatch);
+    const pathName = location?.state?.from?.pathname;
+    pathName ? navigate(pathName) : navigate("/");
```
Comment (biswanath-tewari) [inline src/pages/Auth/Login.jsx:42]: just a small suggestion:
navigate(pathName ? pathName : "/"), this would have been easier to read.

**Review by biswanath-tewari — APPROVED** [review]

File: src/pages/Auth/Login.jsx (hunk 1 above)
Comment (Prabhav-Sharma) [reply src/pages/Auth/Login.jsx:42]: That makes sense, thank you. Made the change

**Pull request merged**